mw-webapp/src/dataAccessLogic/getUsersPreview.ts

```diff
@@ -1,7 +1,6 @@
 import {UserDTOToUserPreviewConverter} from "src/dataAccessLogic/DTOToPreviewConverter/userDTOToUserPreviewConverter";
```

Comment:
Rename and restrucutre file according other models 
For example pattern of file naming - BusinessModelDAL
And inside we use classes

Reply:
Done